_pytest/python.py

```diff
@@ -49,7 +49,10 @@ def _has_positional_arg(func):
 
 
 def filter_traceback(entry):
-    return entry.path != cutdir1 and not entry.path.relto(cutdir2)
+    # ensure entry.path is always a py.path.local object
+    # see https://bitbucket.org/pytest-dev/py/issues/71
+    path = py.path.local(entry.path)
```

Comment:
I think we should move this discussion to the [issue I opened in py](https://bitbucket.org/pytest-dev/py/issues/71), for recording purposes.